nerell-parent/nerell-common/src/main/java/org/arig/robot/config/spring/NerellCommonContext.java

```diff
@@ -143,4 +149,12 @@ public RobotStatus robotStatus() {
     public Ordonanceur ordonanceur() {
         return Ordonanceur.getInstance();
     }
+
+    @Bean(name = "sideServices")
+    public Map<ESide, IRobotSide> sideServices(RightSideService rightSideService, LeftSideService leftSideService) {
+        final Map<ESide, IRobotSide> services = new HashMap<>();
```

Comment:
Quand la clé d'une map est une enum du même type il est préférable d'utilisé une EnumMap

```suggestion
        final Map<ESide, IRobotSide> services = new EnumMap<>(ESide.class);
```